src/lib/libreport-web.sym

```diff
@@ -44,6 +44,7 @@ global:
     abrt_xmlrpc_params_set_value_str;
     abrt_xmlrpc_params_set_value;
     abrt_xmlrpc_new_client;
+    abrt_xmlrpc_new_redhat_client;
```

Comment:
We should also update `-version-info` for libreport-web in `src/lib/Makefile.am`.

Reply:
This libtool magic was always confusing for me. How would you recommend updating the current values?
```
libreport_web_la_LDFLAGS = \
    -version-info 2:1:0
```

Reply:
We have some instructions in the [wiki](https://github.com/abrt/libreport/wiki/Release-Checklist). Following the algorithm, I think the new version should be `3:0:1`.

Reply:
I hope that won't require rebuild of abrt (?) :)

Reply:
You're right. This blog post explains it quite nicely: https://begriffs.com/posts/2021-07-04-shared-libraries.html

I like semver better :)